forage-android/src/main/java/com/joinforage/forage/android/core/services/vault/CheckBalanceRepository.kt

```diff
@@ -40,31 +40,28 @@ internal class CheckBalanceRepository(
             else -> return response
         }
 
-        val vaultResponse = when (
+        val balanceResponse = when (
             val response = vaultSubmitter.submit(
                 getVaultRequestParams(encryptionKeys, paymentMethod)
             )
         ) {
-            is ForageApiResponse.Success -> Message.ModelMapper.from(response.data)
+            // response comes as (snap, non_snap) but we've historically
+            // returned (snap, cash) in our SDK public API. So, we need
+            // to parse the JSON nad convert to (snap, cash) here
```

Comment:
nit: 'and'

Reply:
rut ro I don't think I follow 😭. can you elaborate on the callout?

Reply:
type = fixed! going to mark as resolved!